keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/CtUserResource.java

```diff
@@ -91,29 +93,28 @@ public Response sendMail(EmailModel emailModel) {
             emailModel.setRecipient(user.getEmail());
         }
         if (StringUtils.isBlank(emailModel.getRecipient())) {
-            return ErrorResponse.error(USER_EMAIL_MISSING, Response.Status.BAD_REQUEST);
+            throw ErrorResponse.error(USER_EMAIL_MISSING, Response.Status.BAD_REQUEST);
         }
 
         if (!user.isEnabled()) {
             throw new WebApplicationException(ErrorResponse.error("User is disabled", Response.Status.BAD_REQUEST));
         }
 
-        if (emailModel.getTheming()!=null) {
-            Response error = setEmailTheme(emailModel.getTheming().getThemeRealmName());
-            if (error != null) {
-                return error;
-            }
+        if (emailModel.getTheming() != null) {
+            setEmailTheme(emailModel.getTheming().getThemeRealmName());
```

Comment:
Before we returned an error in case of error, not now is it problematic ?

Reply:
Keycloak changed this too... They throw exceptions in case of error or they return an object (not necessary an instance of Response)